### PR TITLE
Add Japanese translation for "What is OpenTelemetry?"

### PR DESCRIPTION
**Pull request by ymotongpoo**
## Docs PR Checklist

- [X] This PR is for a documentation page whose authoritative copy is in the
      opentelemetry.io repository.

---

**Preview**: https://deploy-preview-4606--opentelemetry.netlify.app/ja/docs/

### CODE REVIEW

**Comment by ymotongpoo** [comment]
This PR failed because of a missing target.

```
> __check:links
> make --keep-going check-links

rm -Rf tmp/bin
curl https://htmltest.wjdp.uk | bash -s -- -b tmp/bin
mkdir -p tmp/.htmltest
cp static/refcache.json tmp/.htmltest/
tmp/bin/htmltest --log-level 1
htmltest started at 02:48:47 on public
========================================================================
running in concurrent mode, this is experimental
ja/docs/index.html
  target does not exist --- ja/docs/index.html --> concepts/observability-primer/#what-is-observability
========================================================================
✘✘✘ failed in 1m2.997326825s
1 errors in [87](https://github.com/open-telemetry/opentelemetry.io/actions/runs/9394342440/job/25871928540?pr=4606#step:4:88)6 documents
cp tmp/.htmltest/refcache.json static/
npm run _prettier:any -- --write static/refcache.json
```

Is this something that https://github.com/open-telemetry/opentelemetry.io/pull/4599 helps?

File: content/ja/docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: ドキュメント
+menu: { main: { weight: 10 } }
+---
+
+OTelの略称でも知られるOpenTelemetryは、[トレース](/docs/concepts/signals/traces/)、[メトリクス](/docs/concepts/signals/metrics/)、[ログ](/docs/concepts/signals/logs/)のようなテレメトリーデータを計装、生成、収集、エクスポートするためのベンダー非依存なオープンソースの[オブザーバビリティ](concepts/observability-primer/#what-is-observability)フレームワークです。
```
Comment (svrnm) [inline content/ja/docs/_index.md:6]:
```suggestion
OTelの略称でも知られるOpenTelemetryは、[トレース](/docs/concepts/signals/traces/)、[メトリクス](/docs/concepts/signals/metrics/)、[ログ](/docs/concepts/signals/logs/)のようなテレメトリーデータを計装、生成、収集、エクスポートするためのベンダー非依存なオープンソースの[オブザーバビリティ](/docs/concepts/observability-primer/#what-is-observability)フレームワークです。
```

Comment (svrnm) [reply content/ja/docs/_index.md:6]: this should fix the issue

**Comment by ymotongpoo** [comment]
@svrnm oh, I did change the path unexpectedly. Thank you for catching the cause!

**Review by svrnm — APPROVED** [review]

**Comment by chalin** [comment]
Thanks all. I have some meetings now, but will come back to this PR. I noticed a few things but don't have time to comment more fully yet, so I've marked this as draft in the meantime.

File: content/ja/docs/what-is-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  OpenTelemetry is an
+  [Observability](/docs/concepts/observability-primer/#what-is-observability)
+  framework and toolkit designed to create and manage telemetry data such as
+  [traces](/docs/concepts/signals/traces/),
+  [metrics](/docs/concepts/signals/metrics/), and
+  [logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
+  tool-agnostic, meaning that it can be used with a broad variety of Observability
+  backends, including open source tools like
+  [Jaeger](https://www.jaegertracing.io/) and
+  [Prometheus](https://prometheus.io/), as well as commercial offerings.
```
[the remark this answers is not in the document]
Comment (ymotongpoo) [reply content/ja/docs/what-is-opentelemetry.md:17]: ah, my bad. thank you for catching.

**Review by chalin — APPROVED** [review]
✨

**Comment by chalin** [comment]
LGTM
FYI, I'll setup fallback pages in a followup PR.

**Comment by chalin** [comment]
@ymotongpoo - note the `default_lang_commit` fields added to the pages of this PR via:

- https://github.com/open-telemetry/opentelemetry.io/pull/4620/files

**Comment by ymotongpoo** [comment]
Thank you @chalin for merging and sharing the commit that adds a revision snapshot of my translation. Can I assume the following PRs will get similar additions by the automated scripts or do I need to add that metadata by myself?

**Comment by chalin** [comment]
Hi @ymotongpoo - the GitHub action only checks whether you've added the necessary commit hash. You'll need to add them yourself based on your knowledge of the commit hash of the English language version of the page(s) you are translating -- which might not be at `main@HEAD`.

If you are working from `en` pages `@HEAD` then you can run the script mention in https://opentelemetry.io/docs/contributing/localization/#adding-default_lang_commit-to-new-pages.

**Comment by ymotongpoo** [comment]
Thank you @chalin for clarifying the process. Then I will add the revisions in the upcoming PRs.

**Pull request merged**